src/Todoist.Net/Services/NotesService.cs

```diff
@@ -23,7 +24,8 @@ public async Task<IEnumerable<Note>> GetAsync(CancellationToken cancellationToke
         {
             var response = await TodoistClient.GetResourcesAsync(cancellationToken, ResourceType.Notes).ConfigureAwait(false);
 
-            return response.Notes;
+            return response.Notes
+                .Concat(response.ProjectNotes);
```

Comment:
Looks like a candidate for a separate PR and discussion

Reply:
OK, what do you suggest then?
Perhaps splitting this method into two? Or returning an object with two collections?

Reply:
> Or returning an object with two collections?

I think this is the good solution.

Reply:
Done, it's on #53 now.